FiveCalls/FiveCalls/PreviewIssues.swift

```diff
@@ -27,4 +27,6 @@ extension Issue {
     
     static let basicPreviewIssue = Issue(id: 813, meta: "", name: "Support the Act", slug: "support-act-slug", reason: Issue.issueReason, script: Issue.issueScript, categories: [Category(name: "Budget")], active: true, outcomeModels: [Outcome(label: "Contacted", status: "contact"), Outcome(label: "Voicemail", status: "voicemail")], contactType: "reps", contactAreas: ["US House", "US Senate"], createdAt: Date(timeIntervalSince1970: 1688015904))
     static let multilinePreviewIssue = Issue(id: 812, meta: "", name: "Support the Act whose name is quite long", slug: "support-act-slug2", reason: Issue.issueReason, script: Issue.issueScript, categories: [Category(name: "Environment")], active: true, outcomeModels: [Outcome(label: "Contacted", status: "contact"), Outcome(label: "Voicemail", status: "voicemail")], contactType: "reps", contactAreas: ["US House", "US Senate"], createdAt: Date(timeIntervalSince1970: 1688015904))
+    static let extraLongPreviewIssue = Issue(id: 812, meta: "", name: "Call for supportive action in a made up region with extra long details", slug: "support-act-slug2", reason: Issue.issueReason, script: Issue.issueScript, categories: [Category(name: "Environment")], active: true, outcomeModels: [Outcome(label: "Contacted", status: "contact"), Outcome(label: "Voicemail", status: "voicemail")], contactType: "reps", contactAreas: ["US House", "US Senate"], createdAt: Date(timeIntervalSince1970: 1688015904))
```

Comment:
the issue id should be distinct from any other preview issues, if you create a  list where items are identified by id and `812` shows up twice SwiftUI starts to behave real weird and it's hard to figure out what's wrong

Reply:
I updated the ID to be unique. We could also consolidate to multilinePreviewIssue only, but use the longer name. I needed the longer name to be a better example of the data that's really out there.